rpcs3/Crypto/unself.cpp

```diff
@@ -5,10 +5,12 @@
 #include "Emu/FS/vfsLocalFile.h"
 #include "unself.h"
 #pragma warning(push)
-#pragma message("TODO: remove wx dependencies: <wx/mstream.h> <wx/zstream.h>")
+#pragma message("TODO: remove wx dependencies: See comment below.")
 #pragma warning(disable : 4996)
-#include <wx/mstream.h>
-#include <wx/zstream.h>
+
+// TODO: Still reliant on wxWidgets for zlib functions. Alternative solutions?
+#include <zlib.h>
+
```

Comment:
I wouldn't mind moving RPCS3 over from wxWidget's zlib to actual zlib, once this and Neko's big PR get merged.